torch/_dynamo/compiled_autograd.py

```diff
@@ -139,6 +139,23 @@ def post_hook(self, outputs, inputs, hook_id):
             self.bind_tensors_to_proxies(outputs, proxies)
         return outputs
 
+    def post_acc_grad_hook(self, inputs, hook_id):
+        assert len(inputs) == 1
+        hook = self.hooks_proxy[hook_id]
```

Comment:
@jansel this is an odd one - if we don't unpack the argument, and leave it to pass in inputs, the hook is invoked with a list, which is incorrect.